webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js

```diff
@@ -444,6 +443,7 @@ export class SakaiRubricCriteria extends RubricsElement {
     const criterion = this.criteria.find(c => c.id == criterionId);
 
     totalPoints += calculateCriteriaPoints(criterion, minOrMax);
+    this.dispatchEvent(new CustomEvent('refresh-total-weight', { detail: { criteria: this.criteria } }));
```

Comment:
Could you confirm this works when deleting a rating, adding a rating, saving a new score for the rating? 

Anyway, I'm going to test these changes locally.

Reply:
It works for the actions you mentioned ✅

Reply:
@V100X Unfortunatelly I still see the issue when deleting a rating, it states 0 points for the criterion. Added a GIF with the issue, when deleting the rating the maximum score of the criterion goes to 0.

![SAK-47640](https://user-images.githubusercontent.com/8653754/184335515-96a953d3-3a35-4519-a00b-e13863352215.gif)

Reply:
Same happens with the min value when removing the lowest rating.
![SAK-47640-2](https://user-images.githubusercontent.com/8653754/184335882-634610a8-a5f1-4b03-901e-d20903f952ff.gif)

Reply:
Works well on my local, anyway, hopefully it's solved now